chingu-voyage4/Geckos-Team-32
# Delete saved videos, fix bugs

## Purpose
* Be able to delete saved videos
* Fix bugs related to styling

## Approach/Fixes
* Re-named `landing` and `postlanding` components to `home` and `search` respectively for more semantic meaning
* Finished backend routing for deleting saved videos
* Added conditional to videos to only show the `heart` icon if logged in
  * Also shows an icon to add video to playlist (clicking it doesn't do anything)
* Added a scroll component to scroll to the top of the window whenever changes pages
* Styled the saved videos for deleting (small x in top-left corner of video card)
* Continued @Raymond-Cox 's code on the frontend to fix the styling issue with the `about` page
* Updated README and CONTRIBUTING docs
* Setup production build in webpack (as far as I know, it works...... LOL)
  * To create the bundle for development run `npm run build:dev`
  * To create the bundle for production run `npm run build:prod`

## Testing
* Download this PR to your local machine
* Run `npm install` to get all the new dependencies
* Test the app on the server with `npm run serve` 
  * Don't forget to run your mongoDB server!!
* Don't forget to comment out the seed data and uncomment the state!!
* Log in and add videos
* Click the `liked videos` link and the news video should appear right away
* Try deleting the saved video (it should disappear instantly)
* Change the window size to all kinds of crazy dimensions and take note of any bugs
